.changeset/kind-clocks-exercise.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+fix: trailing slash on \_\_data.json request
```

Comment:
```suggestion
fix: compute trailing slash for data requests
```

Reply:
added a better description for changeset